spec/controllers/sys_controller_spec.rb

```diff
@@ -211,14 +212,14 @@ module OpenProjectRepositoryAuthenticationSpecs
 
       it 'should return the same as user_login for valid creds' do
         expect(controller.send(:cached_user_login, valid_user.login, valid_user_password)).to eq(
-        controller.send(:user_login, valid_user.login, valid_user_password)
-        )
+                                                                                                controller.send(:user_login, valid_user.login, valid_user_password)
```

Comment:
Line is too long. [163/100]<br>Indent the first parameter one step more than the previous line.